test/fixtures/api-client/assessment_question.findAll.json

```diff
@@ -0,0 +1,184 @@
+{
```

Comment:
there's a mix of case in the filename here is this correct? `api-client/assessment_question.findAll.json`

Reply:
So the thought was to try and dynamically build these in the test cases. It's currently using this:

`fixtures/api-client/{model name}/{devour client method}.json`

I suppose the mix in case is because with the model names we're trying to match the API paths (which use snakecase) and then match the case used by devour for its methods in the second part (camel case).

Maybe we should always stick to using kebab case for filenames and then we can translate to the relevant case in this method.

But then I don't know if that hides the fact that the ultimately these file names need to be correct as they are dynamically looked up.

Reply:
Could add a README to the fixtures folder?

Reply:
> Could add a README to the fixtures folder?

👍 yeah I think this is a good shout to explain what you just have above